builtin/mainmenu/dlg_create_world.lua

```diff
@@ -57,23 +75,6 @@ local flag_checkboxes = {
 		{ "vary_river_depth", fgettext("Vary river depth"),
 		fgettext("Low humidity and high heat causes shallow or dry rivers") },
 	},
-	flat = {
-		cb_caverns,
-		{ "hills", fgettext("Hills") },
-		{ "lakes", fgettext("Lakes") },
-	},
-	fractal = {
-		{ "terrain", fgettext("Additional terrain"),
-		fgettext("Generate non-fractal terrain: Oceans and underground") },
-	},
-	v6 = {
-		{ "trees", fgettext("Trees and jungle grass") },
-		{ "flat", fgettext("Flat terrain") },
-		{ "mudflow", fgettext("Mud flow"), fgettext("Terrain surface erosion") },
-		{ "temples", fgettext("Desert temples"),
-		fgettext("Different dungeon variant generated in desert biomes (only if dungeons enabled)") },
-		-- Biome settings are in mgv6_biomes below
-	},
```

Comment:
The moved code doesn't include all these flags (`temples` was added in a recent commit, I presume)

Reply:
That's the first time I've seen a temples flag, so yes it sounds like a recent commit.